TimeToTrakt.py

```diff
@@ -459,27 +468,356 @@ def processWatchedShows():
+            # to look for a movie with a broadcast year that matches
+            if doesTitleIncludeYear:
+                # If the movie title is a 1:1 match, with the same broadcast year, then bingo!
+                if (name == movie.title) and (movie.year == titleObj.yearValue):
+                    # Clear previous results, and only use this one
+                    moviesWithSameName = []
+                    moviesWithSameName.append(movie)
+                    break
+
+                # Otherwise, only add the movie if the broadcast year matches
+                if movie.year == titleObj.yearValue:
+                    moviesWithSameName.append(movie)
+            # If the program doesn't have the broadcast year, then add all the results
+            else:
+                moviesWithSameName.append(movie)
+
+    # Sweep through the results once more for 1:1 title name matches,
+    # then if the list contains one entry with a 1:1 match, then clear the array
+    # and only use this one!
+    completeMatchNames = []
+    for nameFromSearch in moviesWithSameName:
+        if nameFromSearch.title == name:
+            completeMatchNames.append(nameFromSearch)
+
+    if len(completeMatchNames) == 1:
+        moviesWithSameName = completeMatchNames
+
+    # If the search contains multiple results, then we need to confirm with the user which movie
+    # the script should use, or access the local database to see if the user has already provided
+    # a manual selection
+    if len(moviesWithSameName) > 1:
+
+        # Query the local database for existing selection
+        userMatchedQuery = Query()
+        queryResult = userMatchedMoviesTable.search(
+            userMatchedQuery.movie_name == name)
+
+        # If the local database already contains an entry for a manual selection
+        # then don't bother prompting the user to select it again!
+        if len(queryResult) == 1:
+            # Get the first result from the query
+            firstMatch = queryResult[0]
+            # Get the value contains the selection index
+            firstMatchSelectedIndex = int(firstMatch.get("UserSelectedIndex"))
+            # Check if the user previously requested to skip the movie
+            skipMovie = firstMatch.get("SkipMovie")
+            # If the user did not skip, but provided an index selection, get the
+            # matching movie
+            if not skipMovie:
+                return moviesWithSameName[firstMatchSelectedIndex]
+            # Otherwise, return None, which will trigger the script to skip
+            # and move onto the next movie
+            else:
+                return None
+        # If the user has not provided a manual selection already in the process
+        # then prompt the user to make a selection
+        else:
+            print(
+                f"INFO - MANUAL INPUT REQUIRED: The TV Time data for Movie '{name}' has {len(moviesWithSameName)} matching Trakt movies with the same name."
```

Comment:
For consistency with TV shows from PR #24, a bell output should also be added here

Reply:
Sure, I'll do it.